.github/workflows/docker-publish.yml

```
@@ -64,9 +70,10 @@ jobs:
       # Build and push Docker image with Buildx (don't push on PR)
       # https://github.com/docker/build-push-action
       - name: Build and push Docker image
-        uses: docker/build-push-action@v2
+        uses: docker/build-push-action@v3
         with:
           context: .
+          platforms: linux/amd64,linux/arm/v7
```

If you did armv7, you can also add armv8 to allow raspberry pi 4 to run it.
```suggestion
          platforms: linux/amd64,linux/arm/v7,linux/arm
```